### PR TITLE
chore: update dos games

### PR DESCRIPTION
**Pull request by AustinAbro321**
## Description

This changes the dos-games image to only include doom, and moves the example and e2e tests to using that image.  

## Checklist before merging

- [ ] Test, docs, adr added or updated as needed
- [ ] [Contributor Guide Steps](https://github.com/zarf-dev/zarf/blob/main/CONTRIBUTING.md#developer-workflow) followed

### CODE REVIEW

**Comment by phillebaba** [comment]
One thought I had is that it would be better to put the DOS game demo related things in a separate repository within the org. That way we could automate the release if we want to do anything else in the future.

**Comment by AustinAbro321** [comment]
@phillebaba I agree & I will make an issue for that. One problem with the current setup is that I can't change the usage of the OCI package because it's signed on release with a secret key.  There are some things to think through like if we would still include it in docs in that case. I would still want this PR in first to eliminate the dockerhub flakes.

File: examples/dos-games/image/Dockerfile [hunk 1]
```diff
@@ -34,4 +24,4 @@ COPY --from=0 /binary /binary
 WORKDIR /site
 ENTRYPOINT ["/binary/darkhttpd", "/site", "--port", "8000"]
 
-# docker buildx build --push --platform linux/arm/v7,linux/arm64/v8,linux/amd64 --tag defenseunicorns/zarf-game:multi-tile .
+# docker buildx build --push --platform linux/arm64/v8,linux/amd64 --tag ghcr.io/zarf-dev/zarf-game:0.0.1 .
```
Comment (schristoff) [inline examples/dos-games/image/Dockerfile:27]:
```suggestion
# docker buildx build --push --platform linux/arm64/v8,linux/amd64 --tag ghcr.io/zarf-dev/doom-game:0.0.1 .
```

File: examples/dos-games/manifests/deployment.yaml [hunk 2]
```diff
@@ -13,7 +13,7 @@ spec:
     spec:
       containers:
         - name: multi-game
-          image: "defenseunicorns/zarf-game:multi-tile-dark"
+          image: ghcr.io/zarf-dev/zarf-game:0.0.1
```
Comment (schristoff) [inline examples/dos-games/manifests/deployment.yaml:16]:
```suggestion
          image: ghcr.io/zarf-dev/doom-game:0.0.1
```

File: examples/dos-games/zarf.yaml [hunk 3]
```diff
@@ -14,7 +14,7 @@ components:
           - manifests/deployment.yaml
           - manifests/service.yaml
     images:
-      - defenseunicorns/zarf-game:multi-tile-dark
+      - ghcr.io/zarf-dev/zarf-game:0.0.1
```
Comment (schristoff) [inline examples/dos-games/zarf.yaml:17]:
```suggestion
      - ghcr.io/zarf-dev/doom-game:0.0.1
```

**Review by schristoff — APPROVED** [review]
Great, can you just s/zarf-game/doom-game 
Weird to "claim it is zarf game" when it's just doom

**Comment by phillebaba** [comment]
@AustinAbro321 I am fine with creating an issue and doing it later in the name of removing flakiness in tests due to Docker Hub.

**Comment by AustinAbro321** [comment]
Created  #2853

**Review by schristoff — APPROVED** [review]

**Pull request merged**